script.js

```diff
@@ -0,0 +1,34 @@
+let firstname = document.getElementById("firstname");
+let lastname = document.getElementById("lastname");
+let age = document.getElementById("age");
+let phone = document.getElementById("phone");
+let mail = document.getElementById("mail")
+let male = document.getElementById("male");
+let female = document.getElementById("female");
+let error = document.getElementById("error")
+let sumit = document.getElementById("submit")
+
+sumit.addEventListener("click" , () =>{
+    sumit.innerText="sumited"
+})
+
+function validationInput(input){
+    if(input ==='phone' && phone.value.length !== 10){
+        error.innerText='bhai plz 10 number do'
+    }else if(input === 'age' && age.value < 18){
+        error.innerText="you are not old enough"
+    }else{
+        error.innerText=""
+    }
+}
```

Comment:
create two functions for validating age and phone number,
`function validatePhoneNumber(){
   error.innerText = phone.value.length !== 10 ? "number length should be 10" : "";
}`
`function validateAge(){
   error.innerText = age.value < 18 ? "Your age should be greater or equal to 18" : "";
}`
use these two functions put them at the respective places in html